styles/components/navigation/sideBar.scss

```diff
@@ -173,17 +173,18 @@
             }
         }
 
-        a:focus:not(:focus-visible) {
```

Comment:
I don't think you should remove this.

If you remove it, then the focus ring appears when you click on a navbar item, which is no good.